### PR TITLE
Optimize stub's realloc

### PR DESCRIPTION
**Pull request by MaxGraey**
+ also make `__reset` exportable

### CODE REVIEW

File: std/assembly/rt/pure.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ import { onincrement, ondecrement, onfree, onalloc } from "./rtrace";
 
 // @ts-ignore: decorator
 @global @unsafe
-function __visit(ref: usize, cookie: i32): void {
+export function __visit(ref: usize, cookie: i32): void {
```
Comment (dcodeIO) [inline std/assembly/rt/pure.ts]: This one isn't useful externally

Comment (MaxGraey) [reply std/assembly/rt/pure.ts]: Should I remove declaration from `rt/index.d.ts` as well?

Comment (dcodeIO) [reply std/assembly/rt/pure.ts]: No, internally it is useful (used when implementing RT, which references that file), just not as part of the exported runtime interface.

Comment (MaxGraey) [reply std/assembly/rt/pure.ts]: got it. Done!

**Comment by dcodeIO** [comment]
Looks good, thanks! :)

**Pull request merged**